### PR TITLE
INREL-4974 newsletter styling

### PR DESCRIPTION
**Pull request by Nicozilla**
## [INREL-1234](Ticket-URL) 
We want a clean and simple infinite_theme newsletter template including styling.

## How to test:
Create newsletter content type
- Content > Create new newsletter content

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: js/behaviors/drupal.behaviors.newsletter.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+        alerts.forEach(element => {
+          element.classList.add("remove");
+          setTimeout(() => {
+            element.parentNode.removeChild(element);
+          }, 1000);
+        });
+      },
+      writeAPI() {
+        if (typeof window.thsixtyQ === "undefined") {
+          window.thsixtyQ = window.thsixtyQ || [];
```
Comment (mgoedecke) [inline js/behaviors/drupal.behaviors.newsletter.js]: no need for the check here, you already checked thsixtyQ === undefined

File: js/behaviors/drupal.behaviors.newsletter.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+        if (typeof TrackingManager !== "undefined") {
+          window.TrackingManager.trackEvent(
+            trackingObject,
+            window.TrackingManager.getAdvTrackingByElement(this.$el)
+          );
+
+          if (
+            trackingObject.category === "newsletter" &&
+            trackingObject.action === "success"
+          ) {
```
Comment (mgoedecke) [inline js/behaviors/drupal.behaviors.newsletter.js:213]: no need to track errors as well?

File: templates/node/newsletter/node--newsletter.html.twig [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+ * @ingroup themeable
+ */
+#}
+{# Add wrapper classes #}
+{% set wrapper_classes = [
+'region-full-content',
+'region-infinite-block',
+'newsletter',
+'no-v-padding',
+] %}
```
Comment (mgoedecke) [inline templates/node/newsletter/node--newsletter.html.twig:76]: code style: use indention

File: js/behaviors/drupal.behaviors.newsletter.js (hunk 2 above)
Comment (Nicozilla) [reply js/behaviors/drupal.behaviors.newsletter.js:213]: analog zum alten verhalten

**Review by mgoedecke — APPROVED** [review]
lgtm

**Pull request merged**